functions/index.js

```diff
@@ -250,6 +252,24 @@ async function snapshotData(country) {
     predefinedAcl: "publicRead",
   });
 
+  // Build a simple csv file
+  console.log("Preparing CSV File\n");
+  const csvFileRef = admin.storage().bucket().file(csv_filename);
+  let csv_file_list = [];
+  for (let i = 0; i < data.values.length; i++) {
+      row_string = JSON.stringify(data.values[i]);
```

Comment:
JSON.stringify will use \n and \" for newlines and double quotes. That's not appropriate for csv.